x-pack/plugins/lens/public/visualizations/partition/visualization.tsx

```diff
@@ -275,7 +275,7 @@ export const getPieVisualization = ({
       const originalSecondaryOrder = getSortedGroups(datasource, layer, 'secondaryGroups');
       const accessors = originalSecondaryOrder.map((accessor) => ({
         columnId: accessor,
-        triggerIcon: isCollapsed(accessor, layer) ? ('aggregate' as const) : undefined,
+        triggerIconType: isCollapsed(accessor, layer) ? ('aggregate' as const) : undefined,
```

Comment:
You can also specify here the type to avoid type assertion and have a clean typed map
```
const accessors = originalSecondaryOrder.map<AccessorConfig>((accessor) => ({
        columnId: accessor,
        triggerIconType: isCollapsed(accessor, layer) ? 'aggregate' : undefined,
      }));

Reply:
Fixed [fdea395](https://github.com/elastic/kibana/pull/147686/commits/fdea39540b41cd64635238b285ca35c62e16df1a)